src/custom-apps/components/WebhookDetailsPage/WebhookDetailsPage.tsx

```diff
@@ -92,6 +94,13 @@ const WebhookDetailsPage: React.FC<WebhookDetailsPageProps> = ({
     onSubmit({ ...data, ...{ subscriptionQuery: query } });
   };
 
+  const { data: introspectionData } = useQuery(IntrospectionQuery, {
+    fetchPolicy: "network-only",
+  });
```

Comment:
**issue (blocking):** GraphQL queries should not be in `Page` files, it breaks the convention. Please move it to the parent view component and pass data in props.

Reply:
Fixed